contracts/Governance/AccessControlled.sol

```diff
@@ -50,6 +41,15 @@ abstract contract AccessControlled is Initializable, Ownable2StepUpgradeable {
         return _accessControlManager;
     }
 
+    function __AccessControlled_init(address accessControlManager_) internal onlyInitializing {
+        __Ownable2Step_init();
+        __AccessControlled_init_unchained(accessControlManager_);
+    }
+
+    function __AccessControlled_init_unchained(address accessControlManager_) internal onlyInitializing {
+        _setAccessControlManager(accessControlManager_);
+    }
```

Comment:
Initializer methods are much like constructors, so even though some initializers are internal IMO it makes more sense to put them to the beginning of the contracts. However, if it breaks linters, I'm fine with putting them to internal functions block either. Can we settle on a single approach so that we don't go back and forth with this?